src/main/java/hudson/plugins/git/extensions/impl/CloneOption.java

```diff
@@ -49,13 +59,18 @@ public void decorateCloneCommand(GitSCM scm, Run<?, ?> build, GitClient git, Tas
             listener.getLogger().println("Using shallow clone");
             cmd.shallow();
         }
+        if (noTags) {
+            listener.getLogger().println("Avoid fetching tags");
+            cmd.tags(false);
+        }
         cmd.timeout(timeout);
         cmd.reference(build.getEnvironment(listener).expand(reference));
     }
     
     @Override
     public void decorateFetchCommand(GitSCM scm, GitClient git, TaskListener listener, FetchCommand cmd) throws IOException, InterruptedException, GitException {
-    	cmd.timeout(timeout);
+        cmd.tags(!noTags);
```

Comment:
What GitHub's code code viewer doesn't show here: It were four spaces followed by one tab. The whole file is formatted using spaces, except this one line. I had to add a line right next to it. So I decided to correct it and replaced the tab with four spaces.

My interpretation of "no unrelated formatting changes" was that I should not use auto formatters that change the format all over the file which obfuscates the actual change and makes merging other pull requests touching the same file difficult.